src/equilibrate_utils.jl

```diff
@@ -0,0 +1,726 @@
+
+## args
+ - solver (optional): a subtype of AbstractSteadyStateSolver that specifies how to solve the bifurcation diagram. Default is HCSteadyStateSolver (currently only solver avaiable).
+ - reaction_network: a reaction network.
+ - parameter_values: a vector which specifies the parameter values for which the steady states are to be found. Not required in case the reaction network lacks steady states.
+ - parameter_symbol: the parameter which is to be varried.
+ - parameter_range: a range with all the values of the parameter for which steady states are to be solved.
+
+## kwargs
+ - potential arguments for steady state solvers (however, only current solver does not have any such arguments).
+"""
+function bifurcations_grid(rn::DiffEqBase.AbstractReactionNetwork, args...)
+    return bifurcations_grid(HCSteadyStateSolver(), rn, args...)
+end
+function bifurcations_grid(solver::AbstractSteadyStateSolver, rn::DiffEqBase.AbstractReactionNetwork,p::Vector{Float64},param::Symbol,range::AbstractRange)
+    grid_points = Vector{Union{BifurcationPoint,Nothing}}(fill(nothing,length(range)))
+    for i = 1:length(range)
+        p_i=copy(p); p_i[rn.params_to_ints[param]]=range[i];
+        sol = steady_states(solver, rn, p_i)
+        jac_eigenvals = get_jac_eigenvals(map(s->ComplexF64.(s),sol),param,fill(range[i],length(sol)),rn,p)
+        grid_points[i] = BifurcationPoint(sol,jac_eigenvals,stability_type.(jac_eigenvals))
+    end
+    return BifurcationGrid(param,range,Vector{BifurcationPoint}(grid_points),length(range))
+end
+
+#Generates a 2d grid of bifurcation points, using a given steady state method.
+"""
+    bifurcation_grid_2d([solver], reaction_network, parameter_values, parameter_symbol1, parameter_range1, parameter_symbol2, parameter_range2; kwargs...)
+
+    Ad bifurcation grid, but varries two different parameters. Returning a 2d grid with steady states in each grid point.
+
+## args
+ - solver (optional): a subtype of AbstractSteadyStateSolver that specifies how to solve the bifurcation diagram. Default is HCSteadyStateSolver (currently only solver avaiable).
+ - reaction_network: a reaction network.
+ - parameter_values: a vector which specifies the parameter values for which the steady states are to be found. Not required in case the reaction network lacks steady states.
+ - parameter_symbo1l: the first parameter which is to be varried.
+ - parameter_range1: a range with all the values of the first parameter for which steady states are to be solved.
+ - parameter_symbol2: the second parameter which is to be varried.
+ - parameter_range2: a range with all the values of the second parameter for which steady states are to be solved.
+
+## kwargs
+ - potential arguments for steady state solvers (however, only current solver does not have any such arguments).
+"""
+function bifurcations_grid_2d(rn::DiffEqBase.AbstractReactionNetwork, args...)
```

Comment:
don't need a second function. This is just a different dispatch of `bifurcations_grid`

Reply:
Currently there only exists a single solver to solve steady states. It was written like this to prepare for if/when more was added. But yes, in the meantime we could remove this and have only a single function.